packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -338,7 +392,9 @@ class FlutterErrorBuilder {
   }) {
     addProperty<T>(name, value, level: DiagnosticLevel.error, style: style, linePrefix: linePrefix);
   }
-
+  
+  /// Returns a list of DiagnosticsNode objects including all parts of 
+  /// the error report.
   List<DiagnosticsNode> toDiagnostics() {
```

Comment:
btw this method is a bit of a shortcut that we should perhaps remove and replace with just providing a way to add the contents of one error builder to another.

Reply:
Okay, I'll leave it there for now.